reconstruct/reconstruct.go

```diff
@@ -3,49 +3,49 @@ package reconstruct
 import (
 	"fmt"
 	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/expandedTree"
 	"github.com/vertgenlab/gonomics/fasta"
-	"github.com/vertgenlab/gonomics/tree_newick"
 )
 
 //final function to run
-func Reconstruct(root *tree_newick.NTree, filename_output string) {
-	leaf := Get_leaf(root)
-	branches := Get_branch(root)
+func Reconstruct(root *expandedTree.ETree, outFilename string) {
```

Comment:
Minor, but it might be easier in the long-run if you separate out the reconstruction of the sequence from writing that sequence to a file.  I would have reconstruct return a []*fasta.Fasta instead of returning nothing and writing that []*fasta.Fasta to a filename.